draft-00.xml

```diff
@@ -488,22 +488,16 @@
             <xref target="prop-href">'href'</xref>
             and
             <xref target="prop-format">'format'</xref>.
-            If the 'href' property appears it SHOULD contain
-            a dereferencable URL that points to human-readable
-            text. If the 'format' property appears it SHOULD
-            contain one of the following values:
-            <list style="numbers">
-              <t>"text"</t>
-              <t>"html"</t>
-              <t>"asciidoc"</t>
-            </list>
+            If the 'href' property appears it SHOULD contain a
+            dereferencable URL that points to human-readable text. If
+            the 'format' property appears it SHOULD contain one of the
+            following values: "text", "html", "asciidoc", or "markdown".
```

Comment:
Should this be single quotes around the values for consistency. Seems fairly inconsistent use if you look at the following paragraph. Have not looked throughout the doc, but if you note here there is 'format' and below "format". Probably outside the scope of this PR. @mamund Should we open an issue to make this consistent throughout the spec? Or do that and use single quotes here ITM?